lib/stdlib/test/peer_SUITE.erl

```diff
@@ -563,3 +564,24 @@ docker(Config) when is_list(Config) ->
             ?assertEqual(Node, peer:call(Peer, erlang, node, [])),
             peer:stop(Peer)
     end.
+
+user_crash() ->
+    [{doc, "Test that peer node is halted if peer user process crashes"}].
+
+user_crash(Config) ->
+    Opts = case proplists:get_value(connection, Config) of
+               undefined -> #{name => ?FUNCTION_NAME};
+               Conn -> #{connection => Conn, name => ?FUNCTION_NAME}
```

Comment:
Is there a reason to use `?FUNCTION_NAME` instead of `?CT_PEER_NAME`?

The problem it creates, this test is in 3 different groups, but all 3 use the same peer node name, hence 2 tests can never succeed if the first one fails to shut down the node (they will fail peer node boot).

Reply:
No reason, I just copied that from another testcase. I'll change that.